packages/components/src/theme/ontologybrowser.scss

```diff
@@ -46,13 +50,81 @@
 }
 
 /* PATH INFORMATION */
-.ontology-concept-pathinfo-container {
+.concept-pathinfo-container {
     padding-top: 10px;
+    font-size: 14px;
+
+    .title {
+        font-size: 20px;
+    }
+
+    .current-path-container, .alternate-paths-container {
+        margin-top: 15px;
+
+        .concept-path-container{
+            margin-top: 5px;
+
+            &:hover {
+                background-color: $btn-info-bg;
+                cursor: pointer;
+            }
+        }
+
+        .title {
+            font-size: 14px;
+        }
+    }
 
-    .placeholder {
+    .none-selected, .no-path-info {
         font-size: 20px;
         color: $gray-light;
     }
 }
 
+.concept-path-container {
+    color: $text-color;
+    display: inline-block;
+    padding: 2px 5px 2px;
+    border: solid $gray-no-highlight 1px;
+    border-radius: 2px;
+    font-size: 14px;
 
+    &.selected {
+        background-color: $light-gray;
+    }
+
+    &.collapsed {
```

Comment:
have you reviewed this collapse/expand on hover behavior with Josephine? It is just very different from any interactions we have in LabKey. We would normally put hover info into a tooltip.